src/Maestro/tests/DependencyUpdateErrorProcessor.Tests/DepenedencyUpdateErrorProcessorTests.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using Maestro.Data;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Dotnet.GitHub.Authentication;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.ServiceFabric.Data;
+using Moq;
+using ServiceFabricMocks;
+
+namespace DependencyUpdateErrorProcessor.Tests
+{
+    public class DependencyUpdateErrorProcessorTests : IDisposable
+    {
+        private readonly Lazy<BuildAssetRegistryContext> _context;
+        protected readonly Mock<IHostingEnvironment> Env;
+        protected readonly ServiceProvider Provider;
+        protected readonly IServiceScope Scope;
+        protected readonly MockReliableStateManager StateManager;
+        protected readonly Mock<IRemoteFactory> RemoteFactory;
+
+        public DependencyUpdateErrorProcessorTests()
+        {
+            var services = new ServiceCollection();
+            StateManager = new MockReliableStateManager();
+            RemoteFactory = new Mock<IRemoteFactory>(MockBehavior.Strict);
+            Env = new Mock<IHostingEnvironment>(MockBehavior.Strict);
+            services.AddSingleton(Env.Object);
+            services.AddSingleton<IReliableStateManager>(StateManager);
+            services.AddLogging();
+            services.AddDbContext<BuildAssetRegistryContext>(
+                options => { options.UseInMemoryDatabase("BuildAssetRegistry"); });
+            Provider = services.BuildServiceProvider();
+            services.Configure<GitHubTokenProviderOptions>(
```

Comment:
Don't think I need this, want to verify this once.

Reply:
We synced up, let me know if any questions are still open about this one.

Reply:
okay I will